python/lsst/ts/eas/eas_csc.py

```diff
@@ -108,8 +116,17 @@ async def handle_summary_state(self) -> None:
         self.log.info(f"handle_summary_state {salobj.State(self.summary_state).name}")
         if self.disabled_or_enabled:
             if not self.connected:
+                if self.m1m3_termal_task is not None:
+                    assert self.m1m3_thermal_task is not None  # Get it together mypy
```

Comment:
Are you sure this is really needed? My impression is that `if self.m1m3_termal_task is not None:` would be enough...